.circleci/config.yml

```diff
@@ -6,16 +6,16 @@ version: 2.1
 
 commands:
 
-  py_3_7_setup:
-    description: "Install and switch to Python 3.7.5; also install pip and pytest."
+  py_3_9_setup:
```

Comment:
Why is this change required? Is it necessary for pytorch 2.0?

Reply:
yup, torch 2.0 is [not available](https://app.circleci.com/pipelines/github/pytorch/opacus/2700/workflows/60b99641-92fa-44ad-91c5-5ea3279cd271/jobs/15360) for python3.7 and older (at least on PyPI)